build/version.py

```diff
@@ -2,12 +2,30 @@
 import subprocess
 import sys
 
+
 dirname = os.path.dirname(os.path.abspath(__file__))
-merge_base = subprocess.check_output("git -C {} merge-base HEAD master".format(dirname), shell=True).split()[0]
-major = len(subprocess.check_output("git -C {} log --oneline {}".format(dirname, merge_base), shell=True).splitlines())
 
-if subprocess.check_output("git -C {} rev-parse HEAD".format(dirname), shell=True).split()[0] == merge_base:
-    sys.stdout.write(str(major) + ".0")
+
+def git_output(*cmd):
+    return subprocess.check_output(("git", "-C", dirname) + cmd)
+
+
+def has_master():
```

Comment:
The checkout might not actually have `master` branch (as it is when building non-master on Travis), so I put a check for it which outputs `unknown` as the version string. However I'm not sure if `unknown` is the right version for building a package.